build.gradle

```diff
@@ -162,6 +164,9 @@ configure(javaProjects) {
     }
 
     dependencies {
+        // Testing utilities
+        testCompile project(':testing')
```

Comment:
Nit, it might be more consistent with our ordering by putting this after the if block which adds a compile dependency